packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -8,6 +8,18 @@ import 'package:flutter/painting.dart';
 
 export 'dart:ui' show TextDirection;
 
+/// Determines the politeness setting of aria live in Flutter web.
```

Comment:
I would avoid implying that this API is designed only for Flutter Web. Flutter engine's are replaceable (in fact, the Flutter team maintains 7 of them if I'm counting correctly), and some may support it while others don't. However, in a paragraph below we could explain what we do on the web, e.g. that we use an ARIA live region to cause the browser to produce the announcement.